bin/gltf-pipeline.js

```diff
@@ -1,114 +1,53 @@
 #!/usr/bin/env node
 'use strict';
 var fs = require('fs');
+var argv = require('yargs').argv;
 var path = require('path');
-var argv = require('minimist')(process.argv.slice(2));
-var addDefaults = require('../').addDefaults;
-var removeUnusedImages = require('../').removeUnusedImages;
-var removeUnusedSamplers = require('../').removeUnusedSamplers;
-var removeUnusedShaders = require('../').removeUnusedShaders;
-var removeUnusedTechniques = require('../').removeUnusedTechniques;
-var removeUnusedPrograms = require('../').removeUnusedPrograms;
-var removeUnusedBuffers = require('../').removeUnusedBuffers;
-var removeUnusedBufferViews = require('../').removeUnusedBufferViews;
-var removeUnusedMaterials = require('../').removeUnusedMaterials;
-var removeUnusedSkins = require('../').removeUnusedSkins;
-var removeUnusedCameras = require('../').removeUnusedCameras;
-var removeUnusedTextures = require('../').removeUnusedTextures;
-var removeUnusedMeshes = require('../').removeUnusedMeshes;
-var removeUnusedNodes = require('../').removeUnusedNodes;
-var removeUnusedAccessors = require('../').removeUnusedAccessors;
-var removeUnused = require('../').removeUnused;
-var loadGltfUris = require('../').loadGltfUris;
-var writeGltf = require('../').writeGltf;
-var parseBinaryGltf = require('../').parseBinaryGltf;
-var addPipelineExtras = require('../').addPipelineExtras;
-var convertDagToTree = require('../').convertDagToTree;
-var combineMeshes = require('../').combineMeshes;
-var combinePrimitives = require('../').combinePrimitives;
-var removeUnusedVertices = require('../').removeUnusedVertices;
-var OptimizationStatistics = require('../').OptimizationStatistics;
 var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
 var defined = Cesium.defined;
+var writeGltf = require('../').writeGltf;
+var writeBinaryGltf = require('../').writeBinaryGltf;
```

Comment:
Change these two includes to be like the new ones: `require('../lib/writeGltf');` (unless there was a good reason to keep it the original way)